### ISSUE TITLE
Jaws doesn't read the text-area content on focus to text-area in editing mode.

### ISSUE THREAD

**Issue by niranjanbohra** [FreedomScientific/VFO-standards-support#42]
### Summary

>Jaws with IE11 doesn't read the contents of area vox
> 
> Example:
> 1. https://jsfiddle.net/22pLsujt/
> 2. Tab to go to textbox.

### Expected result

> jaws should read the contents the text-area.

### Actual result

> just reading text-box editing.

### Example

> https://jsfiddle.net/22pLsujt/

**Comment by stevefaulkner** [comment]
codepen test case: https://s.codepen.io/stevef/debug/mpamOb

**Comment by stevefaulkner** [comment]
Verified behavior in JAWS 2018 and NVDA with IE. Seems like it is expected behavior rather than a bug. Will look into it more.

**Comment by stevefaulkner** [comment]
There are some issues with the code, which is causing funky behavior:
**example code:**
`<textarea type="text" name="lastname" value="Mouse"> Hello Textbox, Verify the contents of textbox with Jaws.</textarea>`

Neither the `type` or the `value` attributes are allowed on the `textarea` element.
**recommended code:**
 ```
<label for="test">Textbox</label><br>
<textarea id="test" name="lastname">Hello Textbox, Verify the contents of textbox with Jaws.</textarea>
```

1. non conforming attributes removed
2. `label` used to provide `textarea` with an accessible name

[test case](https://codepen.io/stevef/pen/mpamOb) with code fixed.

JAWS behavior for multiline text boxes is consistent across browsers: accessible name (label), role and instruction is announced when the control receives focus by tab key or by virtual navigation key F.
So for the test case, the following is announced
"Textbox, edit, contains text"

JAWS does not announce the value of a `textarea`, by design.

**Comment by stevefaulkner** [comment]
**Note:** JAWS will announce the text in the text area when reading the entire page "say all" using `insert `+ `down arrow`

**Comment by niranjanbohra** [comment]
Thanks @stevefaulkner , As it's by design, closing the bug.
